hack/render-manifests.go

```diff
@@ -14,25 +14,44 @@ func exitWithError(err error, cause string, args ...interface{}) {
 	os.Exit(1)
 }
 
+const NUM_ARGS = 9
+
+const (
+	HANDLER_NAMESPACE = iota + 1
```

Comment:
I have commented the same at prefix PR, Let's use naming args (knowing correct position for 7 args is problematic).